sourcegraph/sourcegraph
# Remove hover action API

Updated part of https://github.com/sourcegraph/sourcegraph/pull/46059

This PR removes the hover actions API from the code host (browser extension) and legacy blob UI. 

@sqs I see that in https://github.com/sourcegraph/sourcegraph/pull/46059 you removed hover action registration from the browser extension code host logic. I also removed it in this PR. My main question here is this ok to do? I mean, this change would mean that in our browser extension code, the intel tooltip won't have any actions like go to definition, Find references, ... etc. Do we plan to deprecate browser extension too? 


## Test plan
- Check browser extension 
- Check that the new blob UI doesn't have any regressions around the code intel tooltip
